Engine/Src/Ancona/Core2D/Systems/Drawable/FadeDrawableSystem.hpp

```diff
@@ -64,7 +66,7 @@ class FadeDrawableComponent
         /**
          * @brief Speed the fade occurs at.
          */
-        float const FADE_SPEED;
+        float FADE_SPEED = 100.0f;
```

Comment:
Since FADE_SPEED is not a constant, you should make it private and only expose a public getter.